src/Symfony/Component/Message/composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "symfony/message",
+    "type": "library",
+    "description": "Symfony Message Component",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
```

Comment:
I think you can safely add yourself here, authors of the recent workflow and lock components did it.